src/components/dataset/owner-only.vue

```diff
@@ -0,0 +1,134 @@
+<!--
+Copyright 2025 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+<template>
+  <div class="panel panel-simple">
+    <div class="panel-heading">
+      <h1 class="panel-title">{{ $t('panel.title') }}</h1>
+    </div>
+    <div class="panel-body">
+      <form @change="confirmationModal.show()" @submit.prevent>
```

Comment:
I could have put `@change` on each of the individual `<input>` elements, but I thought leveraging event bubbling might be nice.